mmv1/third_party/terraform/resources/resource_bigtable_instance.go

```diff
@@ -107,6 +107,12 @@ func resourceBigtableInstance() *schema.Resource {
 										Required:    true,
 										Description: `The target CPU utilization for autoscaling. Value must be between 10 and 80.`,
 									},
+									"storage_target": {
```

Comment:
Maybe it is too obvious to you; could you explain in the comment what would happen if the per node storage goes below or above the limit?

Reply:
I'm not sure this is the best place to describe how autoscaling works as a feature. We have public docs that describes these https://cloud.google.com/bigtable/docs/autoscaling#parameters

Reply:
perhaps you could expand the autoscaling_config block to reference this the description. It's markdown so you should be able to reference it with `[text](url)`.

Reply:
sorry clarifying to update this in the markdown file only.

Reply:
Yup